packages/migrate/package.json

```diff
@@ -25,13 +26,19 @@
     "@truffle/expect": "^0.1.3",
     "@truffle/interface-adapter": "^0.5.22",
     "@truffle/require": "^2.1.7",
+    "@truffle/resolver": "^9.0.17",
```

Comment:
Is `debug` used in this package?

Reply:
nope, doesn't look like it! I'll get rid of it